lib/utils.js

```diff
@@ -4,10 +4,14 @@ var current_week_number = require('current-week-number');
 var DEFAULT_INDEXES = {};
 var DEFAULT_INTERVALS = {};
 var DEFAULT_TIME_FIELDS = {};
+var DEFAULT_TYPES = {};
+
 var DEFAULT_DEFAULT_READ_INDEX = '*';
 var DEFAULT_DEFAULT_WRITE_INDEX = 'juttle';
 var DEFAULT_DEFAULT_INTERVAL = 'none';
 var DEFAULT_DEFAULT_TIME_FIELD = '@timestamp';
+var DEFAULT_DEFAULT_WRITE_TYPE = 'event';
+var DEFAULT_DEFAULT_READ_TYPE = '';
```

Comment:
As we discussed in person, I find this way of configuring the defaults to be confusing. Also there's no way currently to override these default defaults.